src/test/groovy/io/openliberty/tools/gradle/TestMultiModuleLooseEarWithPages.groovy

```diff
@@ -15,11 +15,12 @@ import javax.xml.xpath.XPathFactory
 import io.openliberty.tools.common.plugins.util.OSUtil
 
 import java.nio.file.Path
+import java.nio.file.Paths
```

Comment:
I don't think you are using `java.nio.file.Path` anymore in this file. You can probably remove the import.